sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -74,221 +57,122 @@
   </SALTelemetry>
   <SALTelemetry>
     <Subsystem>MTMount</Subsystem>
-    <Version>3.2.0</Version>
-    <Author>Ander Ansuategi</Author>
-    <EFDB_Topic>MTMount_Azimuth_Cable_Wrap</EFDB_Topic>
-    <item>
-      <EFDB_Name>ACW_Status</EFDB_Name>
-      <Description>ACW_Status</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>ACW_Status_Drive_1</EFDB_Name>
-      <Description>ACW_Status_Drive_1</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>ACW_Status_Drive_2</EFDB_Name>
-      <Description>ACW_Status_Drive_2</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <EFDB_Topic>MTMount_azimuthDrives</EFDB_Topic>
     <item>
-      <EFDB_Name>ACW_Current_Drive_1</EFDB_Name>
-      <Description>ACW_Current_Drive_1</Description>
+      <EFDB_Name>current1</EFDB_Name>
```

Comment:
Wouldn't it be better to replace all these currents with a single array item?

Reply:
Let's discuss this in parking lot. I also prefer arrays, but scalars have several important advantages:
- The EFD does not handle arrays well. Though it has plenty to deal with and we may come up with a reasonable solution some day.
- The data from the low-level controller is scalars (unfortunately, because it wastes a lot of space: each value has its own name in a json-encoded dict). It is more efficient to not have to preprocess them. When they get back from vacation I will ask them about this, but unless it is trivial for them it's probably not worth changing. A much more important change would be to publish CCW position and velocity; at present one has to know which of the two actuators is running to figure that out (all that data is in the telemetry topic, but it's messy to deal with).